### ISSUE TITLE
DREF Superticket

### ISSUE THREAD

**Issue by anamariaescobar** [IFRCGo/go-frontend#2913]
Changes needed:

**General**

- [ ] Change icon for "Shelter" in all exports
![IFRC-icons-colour_Shelter3 (2)](https://github.com/IFRCGo/go-frontend/assets/69478743/edd22921-a02a-44d2-9698-cd7015b9559e)
- [ ] Add a new option ONLY to Planned interventions called “Coordination and Partnerships” using this icon for the export
![IFRC-icons-colour_Partnership](https://github.com/IFRCGo/go-frontend/assets/69478743/57f90682-606e-4457-b363-6d1539eb8e14)
- [ ] For “Needs and Gaps”, “Planned interventions” and “Actions taken by National Society”, the sectors should follow this order in the export:
> 1. Shelter , housing and settlements
> 2. Livelihoods
> 3. Multipurpose cash
> 4. Health
> 5. Water, Sanitation and hygiene
> 6. Protection, gender and inclusion
> 7. Education
> 8. Migration
> 9. Risk reduction, climate adaptation and recovery
> 10. Community engagement and accountability
> 11. Environmental sustainability
> 12. Coordination and partnerships (only planned interventions)
> 13. Secretariat services (only planned interventions)
> 14. National Society Strengthening (only planned interventions)
- [ ] From Needs and gaps, remove the option for “Shelter Cluster Coordination”
- [ ] Add an option to download “Assessment Report” and “any supporting documentation if relevant (description for the event for imminent)” from the export and from the form. For the export add it as a link, Assessment report at the end of the section "Needs (Gaps) Identified" and supporting documentation at the end of "Description of the event".
- [ ] Move the "Numeric details" from "Essential Information" to “description of the event” under “date of the event”
- [ ] “Submission and contacts” section change name to “Operational timeframes and contacts”
- [ ] Add the following question at the end of “Risk and security considerations” section, “Has the child safeguarding risk analysis assessment been completed?” (Yes/No question) – Should also appear on the exports.
- [ ] For all dates, format should be day-month-year. (For all exports)
- [ ] In section "planned Interventions", on the list of activities, bullet points are only appearing for the first activity
- [ ] Make the content font 11 in size (tittles look ok)
- [ ] Bring back GO logo at the bottom of the pages in the export

**Application**

- [ ] Do not transferred anything from Field Report to ICRC actions.
- [ ] Government has requested international assistance is not transferring from Field Report to DREF application.
- [ ] In “description of the event” section, the field “date of disaster/date when the trigger was met” should appear in the export.
- [ ] In “Current National Society actions” section the field start date of national society actions should appear on the export.
- [ ] Add an option to upload a document below “Crisis category” (optional), field should be named “If available please upload Crisis categorization Analysis”. Make it also available to download in the export 
- [ ] In Previous Operations section, make the hyperlink to the Operational learning dashboard more obvious and change the text to: “To access the Operational Learning Dashboard and check learnings for the country, click HERE”, make it in red if possible.
- [ ] In Previous Operations, if “Has a similar event affected the same area(s) in the last 3 years?” if No, hide the rest of the questions (except lessons learned), if yes, show “Did it affect the same population groups?”, if no, hide the rest of the questions (except lessons learned), and so on with the other 2 questions.
- [ ] In the Plan activities section, transfer the total budget from the Overview (first page).
- [ ] In Event details, Description of the event, after scope and scale of the event add two new fields called “Sources of information (optional) name” and “Sources of information (optional) link” with the possibility to add more (like in the Risk section), with the following guidance below: Add the links and the name of the sources, the name will be shown in the export, as an hyperlink. This should be visible in the export and the sources should be numbered.
- [ ] Needs and Gaps, the field called Any gaps/limitations in the Assessment should not appear if empty.
- [ ] In targeting strategy, add option to upload a file called “Upload any additional support documentation (Optional). Should be able to download in the export (using the link). Any type of file

**Ops Update**

- [ ] When the application was a type “assessment” or “imminent” add a pop out asking if the DREF is changing to response
- [ ] In “Current National Society actions” section the field start date of national society actions should appear on the export.
- [ ] Budget is not appearing in export.
- [ ] In the Timeframes Section, add the Reporting timeframe field, with the option to add to dates. In the export this field will appear in the cover page as shown below:
![image](https://github.com/IFRCGo/go-frontend/assets/69478743/71bf1708-2a6f-44c6-b1ec-175fe6a94110)

**Final Report**

- [ ] Remove the word "New" 
![image](https://github.com/IFRCGo/go-frontend/assets/69478743/8a461bcd-e448-46ad-bd34-1304c19b4266)
- [ ] In the Implementation sections, below Persons Targeted field, add a Field called Persons reached, keep male and female disaggregation as part of the persons reached section. 
![image](https://github.com/IFRCGo/go-frontend/assets/69478743/6471d7e8-6366-4431-a3d4-a31fd416df42)
- [ ] In the Targeting strategy, change the question “Who will be targeted by this operation” to “Who was targeted by this operation” 

Sorry for the loooooong ticket!! :) @udaynwa @frozenhelium @tovari

**Comment by udaynwa** [comment]
@anamariaescobar , thanks a lot for setting up this ticket. We went through the list. Just a couple of questions from us: 

> Add the following question at the end of “Risk and security considerations” section, “Has the child safeguarding risk analysis assessment been completed?” (Yes/No question) – Should also appear on the exports.

Question : 
1. Do we need to add the question to all 3 forms?

> When the application was a type “assessment” or “imminent” add a pop out asking if the DREF is changing to response

Questions : 
1. Show pop-up first time only or show it everytime if the is still 'assessment'?
2. After pop-up, whether to allow users to change the DREF Application type manually or not?

cc @frozenhelium @samshara @nnjemie @tovari

**Comment by anamariaescobar** [comment]
Hello!!
For the child safeguarding, yes, needs to appear in all forms. For the pop up, show it everytime is assessment or imminent (when a new document is added, so show it for every ops update, not needed for final report), yes, allow users to manually change the type.

Thankss!!!

@udaynwa @frozenhelium  @samshara  @nnjemie @tovari

**Comment by anamariaescobar** [comment]
Hey guys! I was just reviewing this, and I have a question, when you asked me about the pop up and if it was needed every time, did you meant every time that specific Ops Update is opened? if that was the question, then no, jus when it is created, but we do need it for every ops update created from an application that is Imminent or Assessment (the user could still manually change it).

@udaynwa @samshara @nnjemie @tovari

**Comment by anamariaescobar** [comment]
@udaynwa, some answers on your questions:

Question: Could you specify where the "Submission and contacts" section is? A screenshot will be perfect.
![image](https://github.com/IFRCGo/go-frontend/assets/69478743/b7c89250-0eca-4691-9ecb-14c00d650273)
Question: Could you provide more details on this? We cannot find the Plan activities section.
Move the “Requested amount in CHF” field from Overview section to Operation section under “Planned Intervention”

@frozenhelium @tovari @nnjemie @samshara

**Comment by udaynwa** [comment]
> Do not transfer anything from the Field Report to ICRC actions.

@anamariaescobar , flagging this here. There is no field with `ICRC actions` in the Field report. There are only `RCRC` and `IFRC` related fields. For not transferring anything from the Field Report, we have done that.

**Comment by anamariaescobar** [comment]
@marcovvu1 can you please take a look and reply ?

**Comment by marcovvu1** [comment]
@samshara  @anamariaescobar  You are right, persons assisted and persons reached are the same, we can keep the concept of those provided with assistance as assisted throughout the whole form. 

Thanks!

**Comment by udaynwa** [comment]
> In Event details, Description of the event, after scope and scale of the event add two new fields called “Sources of information (optional) name” and “Sources of information (optional) link” with the possibility to add more (like in the Risk section), with the following guidance below: Add the links and the name of the sources, the name will be shown in the export, as an hyperlink. This should be visible in the export and the sources should be numbered.

@marcovvu1 @anamariaescobar , the `Scope and scale of event` field exists only in Imminent and Response DREF types. Do we add the to new fields `Sources of information (optional) name` and `Sources of information (optional) link` for all DREF types? Or for Imminent and Response DREF types only?

**Comment by marcovvu1** [comment]
@udaynwa @anamariaescobar Will apply for the DREF for assessment, the sources in this case will be for the question "What happened, where and when"
